openpype/plugins/publish/extract_review_slate.py

```diff
@@ -328,19 +324,25 @@ def process(self, instance):
                 copy_args = (
                     "-metadata",
                     "-metadata:s:v:0",
+                    "-codec:v",
+                    "-pixfmt",
```

Comment:
Args for `codec` and `pixfmt` should be filled same way as here https://github.com/pypeclub/OpenPype/pull/3521/files#diff-1064edbd703275fe1bdff4855b452f266ceef997f708712a16a9a21740653a68R186

Reply:
is the link working?

Reply:
> is the link working?

Seems like it's not. There is `format_args` and `codec_args` calculated from source (approximately line 186). That must be used as source for codec and pixfmt instead.

They actually must be filled otherwise ffmpeg will use default values for the codec and format.